src/storage/v2/indices/text_index.cpp

```diff
@@ -21,21 +20,18 @@
 #include <span>
 #include <vector>
 
+namespace r = ranges;
 namespace memgraph::storage {
 
 std::string GetPropertyName(PropertyId prop_id, NameIdMapper *name_id_mapper) {
   return name_id_mapper->IdToName(prop_id.AsUint());
 }
 
-inline std::string TextIndex::MakeIndexPath(const std::string &index_name) {
+inline std::string TextIndex::MakeIndexPath(std::string_view index_name) {
```

Comment:
I think ideally:
- these should be constants in the unnamed namespace:
```
static constexpr std::string_view kTextIndicesDirectory = "text_indices";

  // Boolean operators that should be preserved in uppercase for Tantivy
  static constexpr std::string_view kBooleanAnd = "AND";
  static constexpr std::string_view kBooleanOr = "OR";
  static constexpr std::string_view kBooleanNot = "NOT";
```
- `text_index_storage_dir_` can also be constant outside the class.
- this then allows you that MakeIndexPath is function in unnamed namespace in cpp file, to avoid polluting TextIndex class